### PR TITLE
LPS-188758 apply verifyProdcutTask only when user set product in workspace project

### PR DESCRIPTION
**Pull request by drewbrokke**
This is an update for [LPS-188758](https://issues.liferay.com/browse/LPS-188758).

### CODE REVIEW

**Comment by drewbrokke** [comment]
@simonjhy any updates on this one? If this looks good to you, we can forward.

**Comment by simonjhy** [comment]
ci:test:gradle-plugins

File: modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/VerifyProductTask.java [hunk 1]
```diff
@@ -29,65 +32,76 @@ public class VerifyProductTask extends DefaultTask {
...
+	public void verifyProduct() throws Exception {
+		WorkspaceExtension.ProductInfo productInfo =
+			_extension.getProductInfo();
 
-	public void setDockerImageLiferay(String dockerImageLiferay) {
-		_dockerImageLiferay = dockerImageLiferay;
-	}
+		if (Objects.isNull(productInfo)) {
+			throw new GradleException(
+				"Can not get produtInfo for product '" + _product + "'");
```
Comment (lawrence-lee) [inline modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/VerifyProductTask.java:60]: @simonjhy and @drewbrokke typo
`produtInfo` should be `productInfo`

Comment (simonjhy) [reply modules/sdk/gradle-plugins-workspace/src/main/java/com/liferay/gradle/plugins/workspace/task/VerifyProductTask.java:60]: I will fix it.

**Pull request closed without merging**